Fixes: #45381 (#50328)
nodejs/node: doc/api/webstreams.md

`node:stream/web` module to be imported.

See: https://nodejs.org/docs/latest-v17.x/api/node:stream/web.html
```

### .toWeb()

The `.toWeb()` method converts a Node.js stream to a web stream. It takes a Node.js stream as input and returns a `ReadableStream` or `WritableStream` object.

The following example shows how to convert a Node.js `Readable` stream to a web `ReadableStream` object:

```js
const { Readable } = require('stream');
const { ReadableStream } = require('node:stream/web');

const nodeStream = new Readable();

const webStream = nodeStream.toWeb();

// Consume the web stream using the async iterator protocol.
for await (const chunk of webStream) {
  console.log(chunk);
}
```

The following example shows how to convert a Node.js `Writable` stream to a web `WritableStream` object:

```js
const { Writable } = require('stream');
const { WritableStream } = require('node:stream/web');

const nodeStream = new Writable();

const webStream = nodeStream.toWeb();

// Write data to the web stream using the async iterator protocol.
await webStream.write('Hello, world!');

// Close the web stream.
await webStream.close();
```

### .fromWeb()

> The `.fromWeb()` method converts a web stream to a Node.js stream. It takes a `ReadableStream` or `WritableStream` object as input and returns a Node.js `Readable` or `Writable` stream.
> 
> The following example shows how to convert a web `ReadableStream` object to a Node.js `Readable` stream:

Reviewer: ```suggestion
The `.fromWeb()` method converts a web stream to a Node.js stream by taking a `ReadableStream` or `WritableStream` object as input and returning a Node.js `Readable` or `Writable` stream.

The following example demonstrates how to convert a web `ReadableStream` object into a Node.js `Readable` stream:
```
